### DISCUSSION TITLE
Static only config without docker socket

### DISCUSSION THREAD

**Discussion by olfway** [PremoWeb/chadburn#93] [General]
Add support to run chadburn with static config and without docker socket

Currently it breaks with:
```
daemon.go:46 ▶ CRITICAL Can't start the app: Get "http://unix.sock/info": dial unix /var/run/docker.sock: connect: no such file or directory
```

**Comment by maietta** [comment]
I'm going to need more information.

Chadburn listens to Docker events via the /var/run/docker.sock file handle. There is no other way that I know of to perform the same task without hitting an API periodically.

A side note, Chadburn uses a 3rd party library to receive updates from the Docker events endpoint which is not real-time. In version 2.x and above, I will be removing this library in favor of directly listening to the Docker Events API which provides real-time updates.

What exactly are you wanting to see implemented?

**Comment by olfway** [comment]
I want to use it as a cron replacement without docker integration

I'll have a static config with jobs like this:
```
[job-local "job-executed-on-current-host"]
schedule = @hourly
command = touch /tmp/example
```

**Comment by maietta** [comment]
Okay, interesting.

So here's the interesting thing about the Chadburn project-- I forked it from a modified version of Ofelia with future plans to make it run in Standalone Docker, Docker Swarm and Kubernetes environments with future support for LXC and other container runtime formats with cross platform support.

Perhaps I can mix a standalone option into the mess. To be honest, this project is both a bit challenging and will be very time consuming but a standalone mode should be quite do-able.

I'll keep this open and provide this as a feature of 2.x which is being worked on now.

**Comment by martadinata666** [comment]
ah another man with same needs as me. https://github.com/PremoWeb/chadburn/issues/5

I hope this will implemented, 👍🏼
